src/sentry/models/organization.py

```diff
@@ -161,6 +161,16 @@ def get_default_owner(self):
             sentry_orgmember_set__organization=self,
         )[0]
 
+    def has_single_owner(self):
+        from sentry.models import OrganizationMember, OrganizationMemberType
+        count = OrganizationMember.objects.filter(
+            organization=self,
+            type=OrganizationMemberType.OWNER,
+            has_global_access=True,
+            user__isnull=False,
+        ).count()
```

Comment:
When I do this, I throw on a `LIMIT 2` since we don't really want to count all the rows, we just want to know if there are 0, 1, or 1+.

Probably not a big deal here though, just a pattern I do.